wkcuber/downsampling.py

```diff
@@ -533,6 +542,28 @@ def detect_larger_and_smaller_dimension(scale):
             args.compress,
             args,
         )
+    elif args.anisotropic:
+        try:
+            scale = read_datasource_properties(args.path)["scale"]
+        except Exception as exc:
+            logging.error(
+                "Could not determine scale which is necessary "
+                "to find target magnifications for anisotropic downsampling. "
+                "Does the provided dataset have a datasource-properties.json file?"
+            )
+            raise exc
+
+        downsample_mags_anisotropic(
+            args.path,
+            args.layer_name,
+            from_mag,
+            max_mag,
```

Comment:
the `max_mag` is isotropic. How do I specify to stop at e.g. 1024-1024-512?

Reply:
The anisotropic parameter decides for itself which mags specifically should be created. It will stop when the current mag is equal or greater than the target mag. For the greater/equal comparison, the maximum dim of the both mags are used. Therefore, you can only control that the "poorest" resolution should not exceed 1024. I added a comment about that to the argparse help (see my latest commit). I hope this is enough for now.